### PR TITLE
Deprecate Istio K8s Service Dashboard

### PR DESCRIPTION
**Pull request by carolynhu**
#28288

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any changes that may affect Istio users.

### CODE REVIEW

**Comment by carolynhu** [comment]
https://snapshot.raintank.io/dashboard/snapshot/4SF1vxUyvxMUaXwRn0mg7BuyOobGXuZ5

File: manifests/addons/dashboards/istio-service-dashboard.json [hunk 1]
```diff
@@ -2984,7 +2984,7 @@
     ]
   },
   "timezone": "",
-  "title": "Istio Service Dashboard",
+  "title": "Istio K8s Service Dashboard (Deprecated)",
```
Comment (bianpengyuan) [inline manifests/addons/dashboards/istio-service-dashboard.json:2987]: This is not necessarily only about k8s. The service can be hosted anywhere.

**Comment by howardjohn** [comment]
I am a bit concerned about the user churn this will cause. Do we need to remove this one and create a new one? Can we just change the existing one to do what we want?

**Comment by carolynhu** [comment]
> I am a bit concerned about the user churn this will cause. Do we need to remove this one and create a new one? Can we just change the existing one to do what we want?

related comments are in https://github.com/istio/istio/pull/29690

I'd prefer, we deprecate this one. and Add a new one, not modifying things based on the old service dashboard. Since those are actually not the same thing, which may make things even more confused.

File: manifests/addons/dashboards/istio-service-dashboard.json (hunk 1 above)
Comment (douglas-reid) [reply manifests/addons/dashboards/istio-service-dashboard.json:2987]: Any suggestions for a better name?  Putting kubernetes isn't perfect, but I'm struggling to think of a better concept here. There is obvious overlap at the edges with the new dashboard.

**Comment by douglas-reid** [comment]
> I am a bit concerned about the user churn this will cause. Do we need to remove this one and create a new one? Can we just change the existing one to do what we want?

We could. Would you prefer a PR that changed this to be canonical services-based content and then added a new separate dashboard for the old content?  That might be a bit more surprising in some cases.

**Comment by howardjohn** [comment]
> We could. Would you prefer a PR that changed this to be canonical services-based content and then added a new separate dashboard for the old content? That might be a bit more surprising in some cases.

Why do we need both?

I bet users already are confused by workload vs service dashboard. I bet most developer will be confused if we have workload, service, and canonical service dashboards

**Comment by douglas-reid** [comment]
> Why do we need both?

We don't, really. K8s services are not necessarily 1:1 with canonical services and the proposed new canonical service dashboard is missing details on breakdown by k8s service, etc., but absent the few details, there isn't a *known* need for multiple dashboards in this space. One issue may be that we don't have a great deal of feedback on the new dashboard yet.

In the design doc, there was a mention of needing to follow proper deprecation policies. I'm not sure that the internals of a dashboard are necessarily bound by that policy, but that was the idea.

I guess we could say that if you want the old dashboard, you can still go get it from grafana.org. Would that be acceptable?

@mandarjog thoughts?

**Pull request closed without merging**